### PR TITLE
Enable inductor CI for TorchBench

### PR DESCRIPTION
**Pull request by desertfire**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #87465

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx

### CODE REVIEW

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -753,6 +771,15 @@ elif [[ "${TEST_CONFIG}" == *inductor* && $SHARD_NUMBER -lt 8 && $NUM_TEST_SHARD
   install_timm
   id=$((SHARD_NUMBER-3))
   test_inductor_timm_shard $id
+elif [[ "${TEST_CONFIG}" == *inductor* && $SHARD_NUMBER -lt 9 && $NUM_TEST_SHARDS -gt 1 ]]; then
+  install_torchaudio
+  install_torchtext
+  install_torchvision
```
Comment (malfet) [inline .jenkins/pytorch/test.sh:777]: Hmm, do you really need to build all 3 of those for all shards?

**Review by malfet — APPROVED** [review]
Looks fine, but before merging this PR, please provide the answers to the following:
- What is the plan for updating audio/text and benchmark pins
- What is the runtime of this newly added 8th shard?


And last but not least, is it just me, or we are trying to rebuild monorepo here? 
And if number of pins are to increase in the future, should we adopt https://gerrit.googlesource.com/gerrit/ ?

**Comment by desertfire** [comment]
> Looks fine, but before merging this PR, please provide the answers to the following:
> 
> * What is the plan for updating audio/text and benchmark pins
> * What is the runtime of this newly added 8th shard?
> 
> And last but not least, is it just me, or we are trying to rebuild monorepo here? And if number of pins are to increase in the future, should we adopt https://gerrit.googlesource.com/gerrit/ ?

Ok, I finally get this working. To answer your questions,

* For audio/text/vision pins, I think once inductor is stable enough, we should just use nightly. For torchbench, we don't expect frequent update, maybe we should still keep its pin? Hopefully this also addresses your question about monorepo.
* For the added torchbench, the test itself is about 35 mins,(https://github.com/pytorch/pytorch/actions/runs/3490880098/jobs/5843200193)

**Pull request closed without merging**